SongBird/app/src/main/java/shuvalov/nikita/songbird/AppData.java

```diff
@@ -0,0 +1,17 @@
+package shuvalov.nikita.songbird;
+
+/**
+ * Created by NikitaShuvalov on 11/17/16.
+ */
+
+public class AppData {
+    public static final String CALLBACK_URL = "http://www.nikitashuvalov.com";
+    public static final String CONSUMER_KEY = "QEqsqDozcgpUZYVsne9Ztdbud";//API-KEY
+    public static final String CONSUMER_SECRET = "zwNkTxCiGIyt2n01wTaOUvUIaUXpKuM3uLBuocRP2ljPvOjN7M";//
+    public static final String ACCESS_TOKEN = "1969838864-I39upLfx5YbjQMSsF4ULAU1v5Q4BaYJZJbbLQXI";
+    public static final String ACCESS_TOKEN_SECRET = "hSboLmM9hHrlGrs1IHhsT7poOaMlYKodnXrmcIMPYinu2";
```

Comment:
I like that you put your constants in a separate file that you can "ignore" from github if you don't want them to be public. Do you need the ACCESS_TOKEN and ACCESS_TOKEN_SECRET constants though? I'm guessing they were leftover from testing.